Eevee.Sleep.Bot/Modules/SlashCommands/ChangeRoleSlashModule.cs

```diff
@@ -0,0 +1,58 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for role things.")]
+public class ChangeRoleSlashModule : InteractionModuleBase<SocketInteractionContext> {
+    [SlashCommand("change", "Select a specific one from the role you have")]
+    [UsedImplicitly]
+    public async Task ChangeRoleAsync() {
+        var roles = UserRoleContoller.FindRoleIdsByUserId(Context.User.Id.ToString());
+        var builder = new ComponentBuilder();
```

Comment:
Refactor the component making process to something similar to `DiscordMessageMaker`

Reply:
like this?
[987db4a](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/987db4a6878953e41d1e50a3b5f89f4b09b61841)